tests/unit/test_addtoken.py

```diff
@@ -0,0 +1,10 @@
+from unittest import mock, TestCase
+from add_token import print_short
+
+class test_at(TestCase):
```

Comment:
test classes are typically named with camel-case: `TestSomething`. Please use a meaningful name for what we test here.

Reply:
My bad, test_at meant test_addtoken but I'll be more clearer.
Thanks